tests/common/factories.py

```diff
@@ -133,3 +133,13 @@ class Meta:  # pylint: disable=no-init, old-style-class
         force_flush = True
 
     userid = factory.LazyAttribute(lambda _: ('acct:' + FAKER.user_name() + '@example.com'))
+
+
+class Setting(ModelFactory):
+
+    class Meta:  # pylint: disable=no-init, old-style-class
+        model = models.Setting
+        force_flush = True
```

Comment:
Is there a reason this has `force_flush = True`? Ideally none of our factories would have this and we'd flush manually in the tests when necessary...